splitio/api/client.py

```diff
@@ -106,11 +107,22 @@ def get(self, server, path, apikey, query=None, extra_headers=None):  # pylint:
                 headers=headers,
                 timeout=self._timeout
             )
-            return HttpResponse(response.status_code, response.text)
+            elapsed = response.elapsed.total_seconds()
+            response = HttpResponse(response.status_code, response.text)
+            self._telemetry_runtime_producer.record_sync_latency(metric_name, elapsed)
+            if not 200 <= response.status_code < 300:
+                self._telemetry_runtime_producer.record_sync_error(metric_name, response.status_code)
+                if metric_name == 'token':
+                    self._telemetry_runtime_producer.record_auth_rejections()
+            else:
+                self._telemetry_runtime_producer.record_suceessful_sync(metric_name, round(1000 * elapsed))
+                if metric_name == 'token':
+                    self._telemetry_runtime_producer.record_token_refreshes()
```

Comment:
token refresh should be tracked here https://github.com/splitio/python-client/blob/master/splitio/push/manager.py#L148

Reply:
also succesful sync for auth